hack/dockerfiles/test.Dockerfile

```diff
@@ -1,6 +1,6 @@
 ARG RUNC_VERSION=74a17296470088de3805e138d3d87c62e613dfc4
 ARG CONTAINERD_VERSION=v1.0.0
-# available targets: buildd (standalone+containerd), buildd-standalone, buildd-containerd
+# available targets: buildd, buildd.oci_worker_only, buildd.containerd_worker_only
```

Comment:
Is `oci_only`, `containerd_only` understandable enough to avoid extra verbosity?